pkg/bootstrap/authenticate.go

```diff
@@ -32,6 +32,8 @@ type VerifyResult struct {
 
 	// InstanceGroupName is the name of the kops InstanceGroup this node is a member of.
 	InstanceGroupName string
+	// InstanceGroupRole is the role of the kops InstanceGroup this node is a member of.
+	InstanceGroupRole string
```

Comment:
Can't kops-controller's common code figure out the role of any given instance group name?